modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,164 +169,165 @@ $ ls $HOME/clusterconfig/
-installer image. This allows you to do bare metal installs with the ISO,
-without requiring access to an HTTP server. See “Embedding an Ignition
-config in the {op-system-first} ISO” for details.
+For ISO installations, instead of passing the location of an Ignition config using a kernel or
+command-line option, you can embed an Ignition config into the ISO installer image. This allows you to do bare metal installs with the ISO, but without requiring access to an HTTP server. See “Embedding an Ignition config in the {op-system-first} ISO” for details.
 
-[id="installation-user-infra-machines-advanced_embedignition_{context}"]
-=== Embedding an Ignition config in the {op-system-first} ISO
-The following procedures describe how to embed an Ignition config into
-the ISO so it is applied when the new installation first boots from disk.
+* **Live install Ignition config**: This type must be created manually and should be avoided if possible, as it is not supported by Red Hat. With this method, the Ignition config passes to the live install medium, runs immediately upon booting, and performs setup tasks before and/or after the {op-system} system installs to disk. This method should only be used for performing tasks that must be performed once and not applied again later, such as with advanced partitioning that cannot be done using a MachineConfig.
++
+For PXE or ISO boots, you can create the Ignition config
+and `APPEND` the `ignition.config.url=` option to identify the location of
+the Ignition config. You also need to append `ignition.firstboot ignition.platform.id=metal`
+or the `ignition.config.url` option will be ignored.
 
-To embed an Ignition config named `worker.ign` into an ISO image
-(for example rhcos-<version>-live.x86_64.iso), copy the image to
-a local directory, then run the coreos-installer container with
-that directory mounted, as follows:
+[id="installation-user-infra-machines-advanced_embedignition_{context}"]
+=== Embedding an Ignition config in the {op-system} ISO
+You can embed an Ignition config at the shell prompt, and then identify it on the `coreos-installer` command line.
 
 .Procedure
 
-. Get the {op-system-first} ISO and Ignition config file and copy them into an accessible directory, such as `/mnt`.
+. Download the `coreos-installer` binary from the following image mirror page: https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/.
+
+. Extract the installer to a local directory.
+
+. Retrieve the {op-system} ISO image and the Ignition config file, and copy them into an accessible directory, such as `/mnt`:
 +
 [source,terminal]
 ----
 # cp rhcos-<version>-live.x86_64.iso bootstrap.ign /mnt/
 # chmod 644 /mnt/rhcos-<version>-live.x86_64.iso
 ----
 
-. Run the following command to run `coreos-installer` from a container to embed the
-Ignition config into the ISO:
+. Run the following command to to embed the Ignition config into the ISO:
```

Comment:
Remove extra "to"
Run the following command to embed the Ignition config into the ISO: